Updating standard parser

vdimikj opened the pull request:
Standard parser is updated by adding "Suite" in context.
Therefore the parser will be able to handle failures which come from Suite context such as this one.

Error: Failed expectation
    at Suite.<anonymous> (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:59:51)
    at step (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:32:23)
    at Object.next (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:13:53)
    at fulfilled (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:4:58)
    at process._tickCallback (internal/process/next_tick.js:103:7)

NickTomlin reviewed (commented):
I think this make sense, thank you!

Could you reformat your commit using semantic release style conventions mentioned in [contributing](https://github.com/NickTomlin/protractor-flake/blob/master/CONTRIBUTING.md)?

E.g. `feat: Update standard parser to accept 'suite'`

After that I can merge it in and it will be automatically released.

Commit:
feat: Update standard parser to accept 'suite'

Standard parser is updated by adding "Suite" in context.
Therefore the parser will be able to handle failures which come from
Suite context such as this one.

Error: Failed expectation
    at Suite.<anonymous> (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:59:51)
    at step (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:32:23)
    at Object.next (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:13:53)
    at fulfilled (/var/www/e2e/P0-keeping-the-data.e2e-spec.ts:4:58)
    at process._tickCallback (internal/process/next_tick.js:103:7)

vdimikj commented:
@NickTomlin  I've changed the commit message to match [contributing](https://github.com/NickTomlin/protractor-flake/blob/master/CONTRIBUTING.md) guide.

NickTomlin reviewed (approved):
💃 thanks!
